src/common/GridList.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components'
+
+const GridList = styled.div`
+  display: grid;
+  grid-template-rows: auto 1fr;
+  height: 100vh;
+  background: linear-gradient(0deg, white 70%, #367860 30%);
+  position: relative;
```

Comment:
You can consider sorting your properties in a specific way (a-z or what the property is about)